appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/InstanceDescriptor.java

```diff
@@ -57,72 +57,72 @@
      * @param obj
      * @return
      */
-    public boolean equals(Object obj);
+    boolean equals(Object obj);
```

Comment:
Update copyright year again

Reply:
Why are these no longer public? I believe these methods are used by other packages.

Reply:
Might be wrong, but I vaguely remembered @arjantijms pointing out in a previous PR of mine that the methods in interfaces are public by default

Reply:
@MattGill98 @smillidge The type is an interface indeed:

```java
public interface InstanceDescriptor
```

Interface methods are public by default and the public access modifier is redundant there.